apps/foundation-demo/src/app/checkout/components/shipping-form/shipping-form.component.scss

```diff
@@ -0,0 +1,91 @@
+@import 'daffodil-util';
+
+.shipping-form {
+
+  &__title {
+    margin-bottom: 15px;
+    @include embolden();
+    @include uppercase();
+  }
+
+  display: grid;
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+  @include section-divider();
+
+  @include breakpoint(large) {
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-areas: 
+      "    title     title      title       title       title       title       title       title       title       title       title       title"
```

Comment:
This seems absurd, what's going on here

Reply:
This happened because in the design the button is a little less than half of the width of whole component, which comes out to about 5/12. I think I could do it one other way with grid, but I think this is the most readable way.